user-documentation/moderne-cli/faq.md

```diff
@@ -40,4 +40,19 @@ mod study . --last-recipe-run --data-table org.openrewrite.table.ParseFailures
 
 If you experience this issue when running `mod git apply`, please try committing/stashing/reverting the unsaved changes before running `mod git apply` again.
 
-If that doesn't work, and you keep running into issues with this, please contact [support@moderne.io](mailto:support@moderne.io) and provide us with the `fix.patch` file and the target files (ideally reduced to the minimum need to replicate the issues).
\ No newline at end of file
+If that doesn't work, and you keep running into issues with this, please contact [support@moderne.io](mailto:support@moderne.io) and provide us with the `fix.patch` file and the target files (ideally reduced to the minimum need to replicate the issues).
+
+## My project only builds resources, not my Maven or other build
+
+If you see `Step 1 - build resources`, then we did not detect any other build tools. Your output would look something like:
+```
```

Comment:
GitHub won't let me suggest this without breaking - but can we add a newline between the `Your output should look something like:` and the three backticks?